include/gps.hpp

```diff
@@ -35,15 +36,10 @@ struct GpsPosition {
     uint8_t satelites;
 };
 
-struct GpsMeasurements {
-    GpsTime time;
-    GpsDate date;
-    GpsPosition position;
-};
-
 Result<Unit, Errc> init(Adafruit_GPS& gps);
-Result<GpsMeasurements, Errc> measure(Adafruit_GPS& gps);
-void print(GpsMeasurements measurements);
+GpsDate read_date(Adafruit_GPS& gps);
+GpsTime read_time(Adafruit_GPS& gps);
+GpsPosition read_position(Adafruit_GPS& gps);
```

Comment:
te nazwy funkcji ciut mi mało mówią. Ok jeśli przekazujemy tu obiekt Adafruit_GPS jako gps ale może zmienić ich nazwy na fetch_gps_date?

Reply:
mozna sobie wydedukowac z czego pobierana jest data na podstawie argumentu, więc umieszczanie slowka gps w nazwie raczej nie jest szczególnie konieczne... 

Wyobraz sobie ze te funkcje to są dodatkowe metody dopisane z zewnatrz klasy, które mają ciupke inny syntax.
```c++
gps.read_date();
read_date(gps);
```

raczej kwestia stylu, wiec mi to tam obojetne

Reply:
w sumie racja, chodziło mi o sytuacje

`read_date(troll);`

i co to mówi? :D

Reply:
read_gps_date(troll) jest równie zastanawiający :D problem pana nazywającego zmienną. Deklaracje funkcji można sobie zawsze podejrzeć i wtedy jest jasne